client/src/firebase/connection.ts

```diff
@@ -18,32 +21,34 @@ const DEV_FIRESTORE_PORT = 8080;
 //   appId: import.meta.env.VITE_FIREBASE_APP_ID
 // };
 const firebaseConfig = {
-  apiKey: "AIzaSyCvoVkNgx38i-WtODOuOqVqbtjFNasbcgY",
-  authDomain: "callforhelp-37002.firebaseapp.com",
-  databaseURL: "https://callforhelp-37002-default-rtdb.firebaseio.com",
```

Comment:
I don't understand in this, just to make sure this attribute is deleted on purpose

Reply:
I'm not sure what you mean by that

Reply:
דיברנו על זה בווטצאפ